### PR TITLE
[FSDE-10980] Refactor of the pipeline to split code in functions

### PR DESCRIPTION
**Pull request by Carabes**

### CODE REVIEW

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-252/2/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://b575810f90b72106d1ca987acc87c7f8718db5c8-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-252-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-252/3/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://c6f9d9def5ab5a9cd05a15a91dd30175fe4d9352-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-252-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-252/4/. :x:

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-252/5/. :white_check_mark:

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-252/6/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://873a54f55c28f47a1b04f96f93478fae2c403e98-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-252-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-252/7/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://add650043836388ee7566dfb6ab1732b9c8e9669-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-252-ux-chi.kubeodc-test.corp.intranet/ (internal)

File: cicd/Jenkins/Jenkinsfile [hunk 1]
```diff
@@ -807,34 +617,7 @@ pipeline {
...
-                jslAwsCliWrapper("s3api put-bucket-policy --bucket ${s3Uri} --policy file://policy.json", 'us-east-1', true)
-                jslAwsCliWrapper("s3api get-bucket-policy --bucket ${s3Uri}", 'us-east-1', true)
-                jslAwsCliWrapper("s3 cp ./dist s3://${s3Uri} --recursive", 'us-east-1', true)
-                jslAwsCliWrapper("s3 website s3://${s3Uri} --index-document index.html --error-document index.html", 'us-east-1', true)
-
-
-                // sh "sleep 10m"
-                // jslAwsCliWrapper("s3 rb ${s3Uri} --force", 'us-east-1', true)
-                echo "This will publish dev build to S3"
+                publishToAssetsServer()
```
Comment (janibolkvadze) [inline cicd/Jenkins/Jenkinsfile:620]:
```suggestion
                publishToS3()
```
Can we change the name of the function to make it more specific?

File: cicd/Jenkins/Jenkinsfile [hunk 2]
```diff
@@ -1186,3 +814,434 @@ def printSummary() {
...
+  env.VERSION = sh (
+    script: """
+      cat package.json | grep '"version":' | cut -f4 -d '"'
+    """,
+    returnStdout: true
+  ).trim()
+  jslHubWrapper("release create -F shortCHANGELOG.md v${env.VERSION}")
+}
+
+def publishToAssetsServer() {
```
Comment (janibolkvadze) [inline cicd/Jenkins/Jenkinsfile:1048]:
```suggestion
def publishToS3() {
```

**Review by janibolkvadze — CHANGES_REQUESTED** [review]

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-252/8/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://ad8e3ea5ead767fbc5dcd5ae82202b9fa8f81725-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-252-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-252/9/. :x:

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-252/10/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://92f4558bed9b02f2fb5ac10b8a2dbc51ded580aa-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-252-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-252/11/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://2c6a8840af32d7084f345d1fb6c92ce7c09f6916-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-252-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-252/12/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://e9c666eb73566c918bfaf7a47fccc51973ea934f-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-252-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-252/13/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://3c03c5e5c6db4089666af1654d4ebfd90880cfd3-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-252-ux-chi.kubeodc-test.corp.intranet/ (internal)

File: cicd/Jenkins/Jenkinsfile (hunk 1 above)
Comment (Carabes) [reply cicd/Jenkins/Jenkinsfile]: Just changed that

File: cicd/Jenkins/Jenkinsfile (hunk 2 above)
Comment (Carabes) [reply cicd/Jenkins/Jenkinsfile]: Just changed that

**Review by janibolkvadze — APPROVED** [review]

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-252/14/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://8ba589b8467511194bc47f6e49b9818c19d18dd2-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-252-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-252/15/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://8a9c63f5d85a8c1cfad94b2c9ee08fb7a14f028a-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-252-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-252/16/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://1b8677f4844cc8dc5cc8ca0eb541be3799ce9405-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-252-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-252/17/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://bc6809338ad7e687c3874153060c09d136158ad9-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-252-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-252/18/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://eed6c18768699c2466b6c65e87504daf32733778-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-252-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-252/20/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://f35c469115c9ca17e639ddc2b86daf45e1cff405-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-252-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Review by jllr — APPROVED** [review]

**Pull request merged**